File: core/feature02_test.go
package main

import (
	"testing"
)

var (
	feature02Names      = []string{"R", "A"}
	feature02Accuracies = []string{"%.0f", "%.2f"}
	feature02Types      = []bool{true, true}
)

func Test01Feature02(t *testing.T) {
	arguments := map[string]interface{}{"K": 200, "G": 300, "D": 100}
	expected := map[string]interface{}{"R": 250, "A": 26.57}
	genericTester(t, feature02(arguments), expected, feature02Names, feature02Accuracies, feature02Types)
}

File: core/feature02_test.js
var feature02Names = ["R", "A"];
var feature02Accuracies = [0, 2];
var feature02Types = [true, true];

function test01Feature02() {
  var arguments = {"K": 200, "G": 300, "D": 100};
  var expected = {"R": 250, "A": 26.57};
  genericTester(feature02_core(arguments), expected, feature02Names, feature02Accuracies, feature02Types);
}

test01Feature02();
